anyway/widgets/urban_widgets/urban_crosswalk_widget.py

```diff
@@ -27,7 +27,9 @@ def generate_items(self) -> None:
         )
 
     @staticmethod
-    def get_crosswalk(yishuv, street, start_time, end_time) -> None:
+    def get_crosswalk(
+        yishuv, street, start_time, end_time
+    ) -> Dict[str, Union[List[Dict[str, Union[int, Any]]], Any]]:

```

Comment:
I think that `Union[List[Dict[str, Union[int, Any]]], Any]` is equivalent to `Any`